src/CodeNamerGo.cs

```diff
@@ -151,7 +151,8 @@ public CodeNamerGo()
                 HttpStatusCode.BadGateway,
                 HttpStatusCode.ServiceUnavailable,
                 HttpStatusCode.GatewayTimeout,
-                HttpStatusCode.HttpVersionNotSupported
+                HttpStatusCode.HttpVersionNotSupported,
+                (HttpStatusCode)429

```

Comment:
Nit it seems like these are clustered by their first digit. That would mean that 429 is currently lumped with the 500s, but it should be with the 400s.

Separately, I've verified that this doesn't seem to be part of .NET's Status enumeration, documentation can be found here: https://msdn.microsoft.com/en-us/library/system.net.httpstatuscode(v=vs.110).aspx

Reply:
Fixed, lololol

Reply:
Lol race conditions